PGM/src/main/java/tc/oc/pgm/polls/types/PollCustom.java

```diff
@@ -0,0 +1,37 @@
+package tc.oc.pgm.polls.types;
+
+import com.google.inject.assistedinject.Assisted;
+import com.google.inject.assistedinject.AssistedInject;
+import org.bukkit.command.CommandSender;
+import tc.oc.commons.core.chat.Audiences;
+import tc.oc.pgm.polls.Poll;
+import tc.oc.pgm.polls.PollManager;
+
+public class PollCustom extends Poll {
+
+    public interface Factory {
+        PollCustom create(String text, CommandSender initiator);
+    }
+
+    private String text;
```

Comment:
Instead of passing a text object, can that just be a method?

`getReasonString()` or `getPollString()`

Reply:
Where would you put the method? PollCustom needs it to be able to display the message they player used.